k8s/upgrades/0.8.0-0.8.1/cstor_pool_post_upgrade.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+function usage() {
+ echo
+ echo "Usage:"
+ echo
+ echo "$0 <spc-name> <openebs-namespace>"
+ echo
+ echo "  <spc-name> Get the SPC name using: kubectl get spc"
+ echo "  <openebs-namespace> Get the namespace where pool pods"
+ echo "    corresponding to SPC are deployed"
+ exit 1
+}
+
+if [ "$#" -ne 2 ]; then
+ usage
+fi
+
+spc=$1
+ns=$2
+retry=false
+
+## Fetching the pod names corresponding to spc
+running_pool_pods=$(kubectl get po -n $ns -l app=cstor-pool,openebs.io/storage-pool-claim=$spc \
```

Comment:
error check here, and in other kubectl commands as well

Reply:
If kubectl command fails script will stop running and it will print error message